res/controllers/Keyboard-Default.kbd.xml

```diff
@@ -0,0 +1,382 @@
+<?xml version="1.0" ?>
+<MixxxKeyboardPreset mixxxVersion="2.0.1+" schemaVersion="1">
+  <info>
+    <name>Default Mapping</name>
+    <author>Mixxx</author>
+    <description>Default mapping for Mixxx.</description>
+  </info>
+  <controller>
+    <group name="[AutoDJ]">
+      <control action="enabled">
+        <keyseq final="1">Shift+F12</keyseq>
+      </control>
+      <control action="fade_now">
+        <keyseq final="1">Shift+F11</keyseq>
+      </control>
+      <control action="shuffle_playlist">
+        <keyseq final="1">Shift+F9</keyseq>
+      </control>
+      <control action="skip_next">
+        <keyseq final="1">Shift+F10</keyseq>
+      </control>
+    </group>
+    <group name="[Channel1]">
+      <control action="LoadSelectedTrack">
+        <keyseq final="1">Shift+Left</keyseq>
+      </control>
+      <control action="back">
+        <keyseq lang="en_US" scancode="30">a</keyseq>
```

Comment:
I like the idea of adding the `<scancode>` element. However, if we add this element, we might as well need a `<modifiers>`. The thing, though, now that there can be two elements per key sequence instead of one, is that we should probably parent them to a `<keyseq>` element, which will support the `lang` attribute, just like the current `<keyseq>` element.

How about:
```XML
<control action="some_action">
  
  <!-- Binds to "Shift+A" key on all not explicit listed layouts. 
  So, this won't work for layouts that lack a key labeled 'A'. -->
  <keyseq>
    <key>a</key>
    <modifiers>Shift</modifiers>
  </keyseq>

  <!-- Binds to "Shift+A" on a "en_US" layout -->
  <keyseq lang="en_US">
    <key>a</key>
    <modifiers>Shift</modifiers>
  </keyseq>

  <!-- Binds to Shift+[key_with_scancode_30] to all not explicit listed layouts -->
  <keyseq>
    <scancode>30</scancode>
    <modifiers>Shift</modifiers>
  </keyseq>
  
  <!-- And your last example, binding to 
  "Ctrl+[key_with_scancode_of_the_'a'_labled_key_on_'en_US']" on 
  all not explicit listed layouts -->
  <keyseq>
    <scancode>
      <key lang="en_US">a</key>
    </scancode>
    
    <modifiers>Shift</modifiers>
  </keyseq>
</control>
```

However, since a scancode says something about a key, not about a key sequence, it feels like `<scancode>` should be a child node of `<key>`, but that might be a bit overkill. What do you think?